### PR TITLE
Feature/weee received copy paste

### PR DESCRIPTION
**Pull request by JB12298**

### CODE REVIEW

File: src/EA.Weee.Web/Areas/AATFReturn/Controllers/ObligatedValuesCopyPasteController.cs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        [HttpGet]
+        public virtual async Task<ActionResult> Index(Guid returnId, Guid aatfId, Guid schemeId)
+        {
+            using (var client = apiClient())
+            {
+                var @return = await client.SendAsync(User.GetAccessToken(), new GetReturn(returnId));
+                var organisationId = @return.ReturnOperatorData.OrganisationId;
+                var viewModel = new ObligatedValuesCopyPasteViewModel()
+                {
+                    AatfId = aatfId,
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/AATFReturn/Controllers/ObligatedValuesCopyPasteController.cs]: Set the organisation id inside here from @return.returnoperatordata.organisationid

File: src/EA.Weee.Web/Areas/Test/Controllers/CopyValuesController.cs [hunk 2]
```diff
@@ -44,13 +44,13 @@ public ActionResult Paste(string returnUrl)
             return View("Paste", null, returnUrl);
         }
 
-        [HttpPost]
-        [ValidateAntiForgeryToken]
-        public ActionResult Paste(string pasteValues, string returnUrl)
-        {
-            TempData["pasteValues"] = pasteProcesser.BuildModel(pasteValues);
+        //[HttpPost]
+        //[ValidateAntiForgeryToken]
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Test/Controllers/CopyValuesController.cs]: Delete the Copy values controller in the test area now as we have an implemented version

File: src/EA.Weee.Web/Areas/Test/Views/CopyValues/Index.cshtml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ﻿@using EA.Weee.Web.Extensions
-@model EA.Weee.Web.Areas.AatfReturn.ViewModels.ObligatedReceivedViewModel
+@model EA.Weee.Web.Areas.AatfReturn.ViewModels.ObligatedViewModel
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Test/Views/CopyValues/Index.cshtml]: Delete this file

File: src/EA.Weee.Web/Areas/Test/Controllers/CopyValuesController.cs (hunk 2 above)
Comment (JB12298) [reply src/EA.Weee.Web/Areas/Test/Controllers/CopyValuesController.cs]: deleted

**Pull request merged**